src/createCycleMiddleware.js

```diff
@@ -1,4 +1,5 @@
 import xs from 'xstream'
+import {adapt} from '@cycle/run/lib/adapt'
```

Comment:
staltz answered. check my latest commit. didn't even test it but i think it should work, right?